roachtest: pgjdbc failed

roachtest.pgjdbc [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7763932?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7763932?buildTab=artifacts#/pgjdbc) on master @ [7823e2f75474417a44f0d34f76c5f8914cb91d52](https://github.com/cockroachdb/cockroach/commits/7823e2f75474417a44f0d34f76c5f8914cb91d52):


```
test artifacts and logs in: /artifacts/pgjdbc/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-749-g7823e2f754
Tests run against pgjdbc REL42.3.3
5792 Total Tests Run
5043 tests passed
749 tests failed
72 tests skipped
178 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: org.postgresql.test.jdbc2.StatementTest.testShortQueryTimeout - unknown (unexpected)
For a full summary look at the pgjdbc artifacts
An updated blocklist (pgjdbcBlocklist) is available in the artifacts' pgjdbc log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92517 roachtest: pgjdbc failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*pgjdbc.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21938

--- a/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
+++ b/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
@@ -31,8 +31,6 @@ var pgjdbcBlockList = blocklist{
 	"org.postgresql.core.OidValuesCorrectnessTest.testValue[oidName=POLYGON, oidValue=604]":                                                                                    "unknown",
 	"org.postgresql.core.OidValuesCorrectnessTest.testValue[oidName=REF_CURSOR, oidValue=1,790]":                                                                               "unknown",
 	"org.postgresql.core.OidValuesCorrectnessTest.testValue[oidName=REF_CURSOR_ARRAY, oidValue=2,201]":                                                                         "unknown",
-	"org.postgresql.core.OidValuesCorrectnessTest.testValue[oidName=TSQUERY, oidValue=3,615]":                                                                                  "unknown",
-	"org.postgresql.core.OidValuesCorrectnessTest.testValue[oidName=TSVECTOR, oidValue=3,614]":                                                                                 "unknown",
 	"org.postgresql.core.OidValuesCorrectnessTest.testValue[oidName=XML, oidValue=142]":                                                                                        "unknown",
 	"org.postgresql.core.OidValuesCorrectnessTest.testValue[oidName=XML_ARRAY, oidValue=143]":                                                                                  "unknown",
 	"org.postgresql.jdbc.BitFieldTest.TestGetObjectForBitFields":                                                                                                               "unknown",

